packages/components/managed-data-table/src/data-table-settings/data-table-settings.spec.js

```diff
@@ -125,18 +126,18 @@ describe('DataTableSettings', () => {
         "[name='table-settings-dropdown']"
       );
 
-      fireEvent.focus(dropDown);
-      fireEvent.keyDown(dropDown, { key: 'ArrowDown' });
-      fireEvent.keyUp(dropDown, { key: 'ArrowDown' });
-      fireEvent.keyDown(dropDown, { key: 'Enter' });
-      fireEvent.keyUp(dropDown, { key: 'Enter' });
+      await fireEvent.focus(dropDown);
+      await fireEvent.keyDown(dropDown, { key: 'ArrowDown' });
+      await fireEvent.keyUp(dropDown, { key: 'ArrowDown' });
+      await fireEvent.keyDown(dropDown, { key: 'Enter' });
+      await fireEvent.keyUp(dropDown, { key: 'Enter' });
```

Comment:
This doesn't return a promise. Does it? https://github.com/testing-library/dom-testing-library/blob/master/types/events.d.ts#L86-L107

https://github.com/testing-library/dom-testing-library/blob/master/src/events.js#L6